rector.php

```diff
@@ -50,8 +48,8 @@
         ConvertImplicitVariablesToExplicitGlobalsRector::class,
     ])
     ->withConfiguredRule(ExtEmConfRector::class, [
-        ExtEmConfRector::PHP_VERSION_CONSTRAINT => '8.2.0-8.3.99',
-        ExtEmConfRector::TYPO3_VERSION_CONSTRAINT => '13.1.0-13.4.99',
+        ExtEmConfRector::PHP_VERSION_CONSTRAINT => '8.2.0-8.99.99',
+        ExtEmConfRector::TYPO3_VERSION_CONSTRAINT => '13.1.0-13.99.99',
```

Comment:
Following you above suggestion, that would be equivalend to 1 13.1?

Reply:
Switched it back. Is the New LTS already decided or is that a plan? From the discussions I hear they sometimes descide to add another version if it is needed for some reason before LTS is released.

Reply:
Don't know, heard no rumours about it. But this extension should serve as a best practise, therefore I suggest to use 13.4.99 instead of 13.99.99. But that is cosmetics. And if it is there really an additional minor release, this can be adjusted accordingly.

Reply:
adjusted it